huaweicloud/resource_huaweicloud_fgs_function_v2.go

```diff
@@ -187,6 +297,27 @@ func resourceFgsFunctionV2Create(d *schema.ResourceData, meta interface{}) error
 	}
 
 	d.SetId(f.FuncUrn)
+	if hasFilledOpt(d, "func_vpc") || hasFilledOpt(d, "mount_config") || hasFilledOpt(d, "app_xrole") ||
+		hasFilledOpt(d, "initializer_handler") || hasFilledOpt(d, "initializer_timeout") {
+		urn := d.Id()
+		if strings.HasSuffix(urn, ":latest") {
+			urn = urn[0 : len(urn)-7]
+		}
```

Comment:
suggest to extract those lines into a function

Reply:
fixed